lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Folders.pm

```diff
@@ -17,17 +17,44 @@
 package OpenQA::WebAPI::Plugin::ObsRsync::Controller::Folders;
 use Mojo::Base 'Mojolicious::Controller';
 use Mojo::File;
+use POSIX 'strftime';
+use Data::Dumper;
 
 sub index {
     my $self   = shift;
-    my $folder = $self->param('folder');
     my $helper = $self->obs_rsync;
-    return undef if $helper->check_and_render_error($folder);
-    my $files
+    my %folders_info;
```

Comment:
That sounds wrong as well but I'm ok with it (in the end one can understand what it is supposed to mean). Not sure why you don't like `folder_info_by_name` or something similar.